### PR TITLE
Ssp2/us 196588/196589 remove buttons for header and footer

### PR DESCRIPTION
**Pull request by gcompton-DFE**
…oter.

### CODE REVIEW

File: DFC.ServiceTaxonomy.CustomEditor/Views/ContentEdit_AdditionalActions.cshtml [hunk 1]
```diff
@@ -13,37 +13,46 @@
     var hasDraft = contentItem.HasDraft();
     var hasEditPermission = await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, contentItem);
```
Comment (rpedu) [inline DFC.ServiceTaxonomy.CustomEditor/Views/ContentEdit_AdditionalActions.cshtml:14]: we dont need to call the authorisation service if the content type is footer or header, can this be done in a if else statment?

File: DFC.ServiceTaxonomy.CustomEditor/Views/ContentsButtonActions_SummaryAdmin.cshtml [hunk 2]
```diff
@@ -25,6 +25,12 @@
     var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, CommonPermissions.PreviewContent, contentItem);
     var previewContentTypes = contentItem.ContentType == ContentTypes.Page || contentItem.ContentType == ContentTypes.JobProfile || contentItem.ContentType == ContentTypes.Footer || contentItem.ContentType == ContentTypes.Header;
 
+    // We do not display the Clone button if the content type is a header or footer.
```
Comment (rpedu) [inline DFC.ServiceTaxonomy.CustomEditor/Views/ContentsButtonActions_SummaryAdmin.cshtml:28]: same as above comment  call the authorisation service if not in any of the content type.

**Review by rpedu — CHANGES_REQUESTED** [review]

File: DFC.ServiceTaxonomy.CustomEditor/Views/ContentEdit_AdditionalActions.cshtml (hunk 1 above)
Comment (gcompton-DFE) [reply DFC.ServiceTaxonomy.CustomEditor/Views/ContentEdit_AdditionalActions.cshtml:14]: Updated to use a coalescing operator instead.

File: DFC.ServiceTaxonomy.CustomEditor/Views/ContentsButtonActions_SummaryAdmin.cshtml (hunk 2 above)
Comment (gcompton-DFE) [reply DFC.ServiceTaxonomy.CustomEditor/Views/ContentsButtonActions_SummaryAdmin.cshtml]: Updated.

**Review by rpedu — APPROVED** [review]

**Pull request merged**